trie/trieStorageManager.go

```diff
@@ -187,6 +187,28 @@ func (tsm *trieStorageManager) Get(key []byte) ([]byte, error) {
 	return tsm.getFromOtherStorers(key)
 }
 
+// GetFromEpoch checks an epoch storer for the given key, and returns it if it is found
+func (tsm *trieStorageManager) GetFromEpoch(key []byte, epoch uint32) ([]byte, error) {
```

Comment:
Now there is some duplication among `trieStorageManager`'s functions (we can fix this).